demisto_sdk/commands/update_xsoar_config_file/tests/update_xsoar_config_file_test.py

```diff
@@ -0,0 +1,250 @@
+import json
+from contextlib import contextmanager
+from pathlib import Path
+from shutil import rmtree
+
+import click
+import pytest
+
+from demisto_sdk.__main__ import xsoar_config_file_update
+from demisto_sdk.commands.common.tools import src_root
+from demisto_sdk.commands.update_xsoar_config_file.update_xsoar_config_file import \
+    XSOARConfigFileUpdater
+
+UNIT_TEST_DATA = (src_root() / 'commands' / 'update_xsoar_config_file' / 'tests' / 'data')
```

Comment:
I'm not familiar with this syntax, can you enlighten me please?

Reply:
It to create a path to the current command files for testing - 
Because we need to test the creation of a file we create a temp direction in every test with this path.